Add the Boost include and library paths; add -fpermissive to fix a compile failure
Patch by Simon Branford (University of Birmingham)
--- Makefile.orig	2022-06-01 14:25:59.350419000 +0100
+++ Makefile	2022-06-10 10:02:39.517207931 +0100
@@ -1,7 +1,7 @@
 #Set inclusion paths here (if boost, bamtools, or args are installed outside your path)
-INCLUDE_DIRS=-ISeqLib -ISeqLib/htslib/
+INCLUDE_DIRS=-ISeqLib -ISeqLib/htslib/ -I${EBROOTBOOST}/include
 #Set library paths here (if boost or bamtools are installed outside your path)
-LIBRARY_PATHS=
+LIBRARY_PATHS=-L${EBROOTBOOST}/lib
 #Set to 0 if you encounter linker errors regarding strings from the bamtools library
 ABI=1
 #Provide full paths here to .a archives for libraries which should be statically linked
@@ -11,7 +11,7 @@
 
 CC=g++
 STDLIB=-std=c++14
-CFLAGS=-Wall $(STDLIB) -D_GLIBCXX_USE_CXX11_ABI=$(ABI) -O3
+CFLAGS=-Wall $(STDLIB) -D_GLIBCXX_USE_CXX11_ABI=$(ABI) -O3 -fpermissive
 SOURCES=BED.cpp Expression.cpp GTF.cpp RNASeQC.cpp Metrics.cpp Fasta.cpp BamReader.cpp
 SRCDIR=src
 OBJECTS=$(SOURCES:.cpp=.o)
